tests/unit_tests/kotahi_docmaps/v1/test_data.py

```diff
@@ -93,122 +93,122 @@
 COLON = ':'
 
 REVIEW_1 = f'''
```

Comment:
[textwrap.dedent](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) is what we can use to remove the identation.

e.g. in this case:

```python
REVIEW_1 = textwrap.dedent(
    f'''
    Reviewer #1 (Public Review){COLON}{SPACE}
    This is review_1.
    '''
)
```

The example also adds a `\` after the `'''` to avoid an empty line at the beginning. Not sure if we strip that anyway. But we could do that too (or just call `lstrip` I suppose).

e.g.

```python
REVIEW_1 = textwrap.dedent(
    f'''\
    Reviewer #1 (Public Review){COLON}{SPACE}
    This is review_1.
    '''
)
```